documentation/src/main/asciidoc/reference/coordination.asciidoc

```diff
@@ -0,0 +1,508 @@
+when <<coordination-database-polling-processors,processing is disabled>>.
+====
+
+.Example of static sharding settings
+====
+For example, the following configuration with 4 application nodes
+would assign shard `0` to application node #0, shard `1` application node #1,
+and no shard at all to application nodes #2 and #3:
+
+[source]
+----
+# Node #0
+hibernate.search.backend.coordination.strategy = database-polling
+hibernate.search.backend.coordination.shards.static = true
+hibernate.search.backend.coordination.shards.total_count = 2
+hibernate.search.backend.coordination.shards.assigned = 0
+----
+
+[source]
+----
+# Node #1
+hibernate.search.backend.coordination.strategy = database-polling
+hibernate.search.backend.coordination.shards.static = true
+hibernate.search.backend.coordination.shards.total_count = 2
+hibernate.search.backend.coordination.shards.assigned = 1
+----
+
+[source]
+----
+# Node #2
+hibernate.search.backend.coordination.strategy = database-polling
+hibernate.search.backend.coordination.processors.indexing.enabled = false
+----
+
+[source]
+----
+# Node #3
+hibernate.search.backend.coordination.strategy = database-polling
+hibernate.search.backend.coordination.processors.indexing.enabled = false
+----
+====
+
+[TIP]
+====
+Sharding settings, even static, may change over the lifetime of an application,
+but must be consistent across all application nodes involved in processing at all times.
+
+If you cannot stop your whole application cluster,
+the only way to safely change sharding settings is to restart nodes one by one
+to disable <<coordination-database-polling-processors,processing>>,
+then (once every node has processing disabled),
+restart nodes one by one with the new sharding settings and with processing re-enabled.
+====
+
+[[coordination-database-polling-processors]]
+=== Processors
+
+The database-polling coordination strategy involves background processors,
+in particular one that polls the outbox events table for events
+and then reindex the corresponding entities when new events are found.
+
+These processors can be configured using the following configuration properties:
+
+[source]
+----
+hibernate.search.backend.processors.indexing.enabled = true
+hibernate.search.backend.processors.indexing.polling_interval = 100
+hibernate.search.backend.processors.indexing.batch_size = 50
+----
+
+* `processors.indexing.enabled` defines whether the background processor responsible for indexing entities is enabled,
+as a <<configuration-property-types,boolean value>>.
+The default for this property is `true`, but it can be set to `false` to disable background processing on some application nodes,
+for example to dedicate some nodes to HTTP request processing and other nodes to background processing.
+* `processors.indexing.polling_interval` defines how long to wait for another query to the outbox events table
+after a query didn't return any event,
+as an <<configuration-property-types,integer value>> in milliseconds.
```

Comment:
I would add that this property value may affect the time needed to see changes on indexes